src/test/java/com/google/api/generator/gapic/composer/grpcrest/goldens/HttpJsonEchoStub.golden

```diff
@@ -78,7 +78,7 @@ public class HttpJsonEchoStub extends EchoStub {
                       })
                   .setRequestBodyExtractor(
                       request ->
-                          ProtoRestSerializer.create().toBody("*", request.toBuilder().build()))
+                          ProtoRestSerializer.create().toBody(request.toBuilder().build(), false))
```

Comment:
I agree it is a bit subjective, but can you please keep the field name thing, as it looks clearer and matches the proto config more directly (config does have the names so it is easier to search and match things between generated code and proto file). Also it is basically an implementaiton detail that the field name is not actually used now, it may be used later.

Reply:
Added the fieldName back.